sdk/src/test/java/com/karhoo/sdk/api/service/trips/CancellationFeeInteractorTest.kt

```diff
@@ -0,0 +1,123 @@
+package com.karhoo.sdk.api.service.trips
+
+import com.karhoo.sdk.api.KarhooError
+import com.karhoo.sdk.api.KarhooSDKConfigurationProvider
+import com.karhoo.sdk.api.model.AuthenticationMethod
+import com.karhoo.sdk.api.model.BookingFee
+import com.karhoo.sdk.api.model.BookingFeePrice
+import com.karhoo.sdk.api.network.response.Resource
+import com.karhoo.sdk.api.testrunner.UnitTestSDKConfig
+import com.karhoo.sdk.api.testrunner.base.BaseKarhooUserInteractorTest
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.whenever
+import junit.framework.Assert.assertEquals
+import junit.framework.Assert.assertNotNull
+import kotlinx.coroutines.CompletableDeferred
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.mockito.ArgumentMatchers
+
+class CancellationFeeInteractorTest : BaseKarhooUserInteractorTest() {
+
+    private lateinit var interactor: CancellationFeeInteractor
+
+    @Before
+    override fun setUp() {
+        super.setUp()
+        whenever(credentialsManager.isValidToken).thenReturn(true)
+        interactor = CancellationFeeInteractor(credentialsManager, apiTemplate, context)
+    }
+
+    /**
+     * Given:   A valid trip id
+     * When:    When requesting to cancel a trip
+     * Then:    The cancellation fee should come back with true
+     */
+    @Test
+    fun `cancellation to a trip returns true`() {
```

Comment:
Might be worth adding that this has no fee to enable easier differentiation with the next test